codegen/masm/src/codegen.rs

```diff
@@ -287,6 +365,140 @@ impl<'ast> CodeGenerator<'ast> {
         Ok(())
     }
 
+    /// Emits code for the procedure `get_exemptions_points`.
+    ///
+    /// The generated procedure contains the precomputed exeption points to be used when computing
```

Comment:
minor nit: exception

Reply:
or exemption (first time I notice they are close in Hamming distance :smile:)

Reply:
fixed